src/vm.js

```diff
@@ -494,16 +497,14 @@ class VM extends common.ServiceObject {
    *   const apiResponse = data[1];
    * });
    */
-  getSerialPortOutput(port, callback) {
-    if (is.fn(port)) {
-      callback = port;
-      port = 1;
-    }
+  getSerialPortOutput(optionsOrCallback, cb) {
```

Comment:
Also it seems they are not using `optionsOrCallback` approach in `.js` files. rather just `(param, options, callback)` and then
```ts
    if (is.fn(options)) {
      callback = options;
      options = {};
    }
    options = options || {};
```